typescript/hx-adl/src/gen-sqlschema.ts

```diff
@@ -10,45 +10,59 @@ import { snakeCase } from "change-case";
 
 export function configureCli(program: Command) {
   program
-   .command("sql [adlFiles...]")
-   .option('-I, --searchdir <path>', 'Add to adl searchpath', collect, [])
-   .option('--outfile <path>', 'the resulting sql file', 'create.sql')
-   .option('--outputdir <dir>', 'the directory into which the sql is written (deprecated)')
-   .option('--outmetadata <path>', 'sql to insert the model metadata')
-   .option('--outtemplatesql <paths>', 'generate extra sql from a mustache template', collect, [])
-   .option('--postgres', 'Generate sql for postgres')
-   .option('--postgres-v2', 'Generate sql for postgres (model version 2)')
-   .option('--mssql', 'Generate sql for microsoft sqlserver')
-   .option('--extension <ext>', 'Add to included sql extensions', collect, [])
-   .description('Generate a db schema from ADL files')
-   .action( (adlFiles:string[], cmd:{}) => {
-     const adlSearchPath: string[] = cmd['searchdir'];
-     const extensions: string[] = cmd['extension'];
-     const templates: Template[] = parseTemplates(cmd['outtemplatesql'] || []);
-
-     let outfile: string = cmd['outfile'];
-     if (cmd['outputdir']) {
-       outfile = cmd['outputdir'] + '/create.sql';
-     }
-
-     let outmetadata: string | null = cmd['outmetadata'] || null;
-
-     let dbProfile = postgresDbProfile;
-     if (cmd['postgresV2']) {
-       dbProfile = postgres2DbProfile;
-     }
-     if (cmd['mssql']) {
-       dbProfile = mssql2DbProfile;
-     }
-
-     generateSql({adlFiles, adlSearchPath, outfile, outmetadata, extensions, templates, dbProfile});
-   });
+    .command("sql [adlFiles...]")
```

Comment:
This file shows a lot of whitespace changes - It looks like you probably used auto-prettier on save.

You can use https://rnjn.in/2015/09/cleaning-up-whitespace-changes-in-a-pull-request.html to undo unintentional whitespace changes.

Whitespace changes (even if actually tidy-ups) obscure the changes and it's better to do them in separate changes, especially if there's lots of them across the file.

When committing files it's a good idea to self review which changes are actually going into a commit (I use git gui personally).